src/shared/bg_public.h

```diff
@@ -1127,10 +1127,24 @@ enum meansOfDeath_t
 // TODO: move other button definitions into gamelogic
 enum buttonNumber_t
 {
-  BUTTON_ATTACK3 = 9,
-  BUTTON_DECONSTRUCT = 13,
+	BTN_ATTACK       = 0,
+	// defined by daemon
+	BTN_TALK         = BUTTON_TALK, // = 1, // disables actions
+	BTN_USE_holdable = 2,
+	BTN_GESTURE      = 3,
+	// defined by daemon
+	BTN_WALKING      = BUTTON_WALKING, // = 4, // walking can't just be inferred from MOVE_RUN
+	BTN_SPRINT       = 5,
+	BTN_ACTIVATE     = 6,
+	// defined by daemon
+	BTN_ANY          = BUTTON_ANY, // = 7, // if any key is pressed
+	BTN_ATTACK2      = 8,
+	BTN_ATTACK3      = 9,
+	BTN_DECONSTRUCT  = 13,
+	BTN_RALLY        = 14 // also named "taunt" in some places
```

Comment:
(nitpick) Typically one adds a comma to help limiting future diffs
```suggestion
	BTN_RALLY        = 14, // also named "taunt" in some places
```